scripts/imgtool/image.py

```diff
@@ -275,6 +279,12 @@ def create(self, key, enckey, dependencies=None):
             # Add the size of the TLV info header
             protected_tlv_size += TLV_INFO_SIZE
 
+        if custom_protected_tlv:
+            if protected_tlv_size == 0:
+                protected_tlv_size = TLV_INFO_SIZE
+            for tag, value in custom_protected_tlv.items():
```

Comment:
If passing a tuple you could use `for _, (length, _) in custom_tlvs.items()`, and the size would be `TLV_SIZE + length`